modules/autofill/autofill-api/src/main/java/com/duckduckgo/autofill/InternalTestUserChecker.kt

```diff
@@ -16,8 +16,27 @@
 
 package com.duckduckgo.autofill
 
+/**
+ * Public API that could be used if the user is a verified internal tester.
+ * This class could potentially be moved to a different module once there's a need for it in the future.
+ */
 interface InternalTestUserChecker {
+    /**
+     * This checks if the user has went through thr process of becoming a verified test user
+     */
     val isInternalTestUser: Boolean
+
+    /**
+     * This method should be called will be called if an error is received when loading a [url].
```

Comment:
Nit: "This method should be called will be called "   should be called / will be called (pick one?)